### ISSUE TITLE
Convert Conditions to Code

### ISSUE THREAD

**Issue by ardeay** [EchoMTG/mtg-csv-reader#11]
Convert full name conditions to their code

``` let conditions = {
                'Near Mint' :  'NM',
                'Lightly Played'  : 'LP',
                'Moderately Played' : 'MP',
                'Heavily Played' : 'HP',
                'Damaged' : 'D',
                'Altered' : 'ALT',
                'Artist Proof' : 'ART',
                'Pre-release' : 'PRE',
                'Timestamped' : 'TS',
                'Signed' : 'SGN',
                'BGS' : 'BGS',
                'BGS 10' : 'B10',
                'BGS 9.5' : 'B95',
                'BGS 9.0' : 'B9',
                'BGS 8.5' : 'B85',
                'BGS 8.0' : 'B8',
                'BGS 7.5' : 'B75',
                'BGS 7.0' : 'B7',
                'PSA' : 'PSA',
                'PSA 10' : 'P10',
                'PSA 9.5' : 'P95',
                'PSA 9.0' : 'P9',
                'PSA 8.5' : 'P85',
                'PSA 8.0' : 'P8',
                'PSA 7.5' : 'P75',
                'PSA 7.0'   : 'P7'
            };```

**Comment by Jed-Giblin** [comment]
This was started in e43b3d8ced1a1782f3690286ca00c60e91e8734a

**Comment by Jed-Giblin** [comment]
This was resolved in #13
